apps/frontend/src/pages/Build/BuildWorkspace.tsx

```diff
@@ -108,7 +109,7 @@ export const BuildWorkspace = (props: {
               build && (
                 <>
                   <BuildDetail build={build} repoUrl={repoUrl} />
-                  {build.type === "orphan" && (
+                  {build.type === "orphan" && build.status === BuildStatus.DiffDetected && (
```

Comment:
I don't understand this line. Can you explain why did you add this?

Reply:
If already reviewed, no dialog.

Reply:
DiffDetected !== already reviewed

Reply:
That's why it's a === and not a !==